### PR TITLE
Re IDC #2604: show on slides panel and on tag explorer series which are non supported for the visualization.

### PR DESCRIPTION
**Pull request by Punzo**
Implements IDC task: https://github.com/OHIF/Viewers/issues/2604

idc2604
idc2604_tag

@igoroctaviano can you please review and test? Thanks!

study: https://viewer.imaging.datacommons.cancer.gov/viewer/1.3.6.1.4.1.14519.5.2.1.2103.7010.333899005177230305399406859716

### CODE REVIEW

**Review by igoroctaviano — CHANGES_REQUESTED** [review]
Tested and works fine. 👍 
I left some minor comments.

File: platform/core/src/classes/StudyPrefetcher.js [hunk 1]
```diff
@@ -94,7 +92,11 @@ export class StudyPrefetcher {
     }
 
     this.element = element;
-    this.enabledElement = cornerstone.getEnabledElement(element);
+    try {
+      this.enabledElement = cornerstone.getEnabledElement(element);
```
[the remark this answers is not in the document]
Comment (Punzo) [reply platform/core/src/classes/StudyPrefetcher.js:96]: I agree, but I would open a new issue/PR for this.

File: platform/viewer/src/connectedComponents/ConnectedStudyBrowser.js [hunk 2]
```diff
@@ -59,10 +59,14 @@ const mapDispatchToProps = (dispatch, ownProps) => {
...
 
-        if (!displaySet) {
-          throw new Error('Source data not present');
-        }
+      if (!displaySet) {
+        throw new Error('Source data not present');
+      }
+
+      if (!displaySet.isModalitySupported) {
+        throw new Error('Modality not supported');
```
[the remark this answers is not in the document]
Comment (Punzo) [reply platform/viewer/src/connectedComponents/ConnectedStudyBrowser.js:69]: yes you are right!

**Comment by Punzo** [comment]
> Tested and works fine. +1 I left some minor comments.

Perfect! thanks for testing and the review. I have applied in https://github.com/OHIF/Viewers/pull/2621/commits/e070aa72b00411186e7d5d5f5f93cd79546a41f4. I will merge now

**Pull request merged**